dev/Repeater/ItemTemplateWrapper.cpp

```diff
@@ -55,6 +55,11 @@ winrt::UIElement ItemTemplateWrapper::GetElement(winrt::ElementFactoryGetArgs co
         // no element was found in recycle pool, create a new element
         element = selectedTemplate.LoadContent().as<winrt::FrameworkElement>();
 
+        // Template returned null, so insert empty element to render nothing
+        if (!element) {
+            element = winrt::XamlReader::Load(L"<Rectangle xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation' Height='0' Width='0' />").as<winrt::Rectangle>();
```

Comment:
Just creating a rectangle in code would be more efficient than parsing the xaml like so...
element = winrt::Rectangle();
element.Width =  0;
element.Height = 0;

Reply:
I could not figure out how to typecast correctly, since element is UIElement meaning `Heigh` and `Width` are unavailable. :/

Reply:
you will need to cast it to FrameworkElement

Reply:
Changed that with latest commit.